src/views/v2/attendee-registration/content/notice.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Attendee registration
+ * Notice
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/attendee-registration/content/notice.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1amp
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+// @todo @juanfra: componetize v2 for the notice.
+$this->template(
+	'components/notice',
+	[
+		'id'             => 'tribe-tickets__notice__attendee-registration',
+		'notice_classes' => [
+			'tribe-tickets__notice--error',
+			'tribe-tickets__validation-notice',
+		],
+		'content'        => sprintf(
+			// Translators: %s HTML wrapped number of tickets.
+			esc_html_x(
+				'You have %s ticket(s) with a field that requires information.',
+				'Note about missing required fields, %s is the html-wrapped number of tickets.',
+				'event-tickets'
+			),
+			'<span class="tribe-tickets__notice--error__count">1</span>'
+		),
+	]
+);
```

Comment:
Thinking of the type of call, which is a notice, I preferred to make it an independent template. We already have this type of thing in other places. And in case somebody wants to add a class to the notice (quite likely when customizing) they don't have to review the template changes whenever we make changes/update `content.php` (most likely than updating the notice).

Reply:
I think this is fine either way. But we should be consistent. I've seen a notice like this called both in another template and it's own template file.

@bordoni Which direction do you think we should go?

Notices in the ET/ET+ v2 templates include:

* `src/views/v2/attendee-registration/content/notice.php` (separate)
* `src/views/v2/attendee-registration/footer.php` (included with other templating)
* `src/views/v2/modal/attendee-registration/notice/error.php` (separate)
* `src/views/v2/modal/attendee-registration/notice/non-ar.php` (separate)
* `src/views/v2/tickets/notice.php` (separate)